portfolio/src/main/webapp/index.html

```diff
@@ -165,11 +165,35 @@ <h5>What? — May 27, 2020</h5>
     </div>
   </div>
   <div class="card-view">
-    <form action="/comments" method="POST">
+    <label for="select-max-numbers">Select maximum number of
+      comments to display:</label>
+    <select id="max-number-display" name="max-numbers"
+      onchange="populateComments()">
+      <option value="5">5</option>
+      <option value="10">10</option>
+      <option value="15" selected>15</option>
+      <option value="20">20</option>
+      <option value="25">25</option>
+      <option value="100">100</option>
```

Comment:
Curious if you actually tested this ;)

Reply:
I did up to 20 and thought the rest would work ;i